### PR TITLE
Add CBC logo and info

### PR DESCRIPTION
**Pull request by cbccombine**
BSC-CBC-
0xa348C8A0E93d97Fbd2629448eCB4CacD6EB2b953
https://bscscan.com/tx/0xcc309037dfcd8e19f451d5846312a96b485fc642005c32db51d3a3c7f2790d7e

### CODE REVIEW

File: 0xa348C8A0E93d97Fbd2629448eCB4CacD6EB2b953/info.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+"name": "TokenPocket Token",
+"website": "https://tokenpocket.pro",
+"description": "A leading multi-chain wallet that supports BTC, ETH, BSC, HECO, TRON, EOS, OKExChain, HSC and so on.",
+"explorer": "https://bscscan.com/token/0xa348C8A0E93d97Fbd2629448eCB4CacD6EB2b953",
+"type": "BEP20",
+"symbol": "TPT",
```
Comment (cbccombine) [inline 0xa348C8A0E93d97Fbd2629448eCB4CacD6EB2b953/info.json:7]: "symbol": "CBC",

File: 0xa348C8A0E93d97Fbd2629448eCB4CacD6EB2b953/info.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+"name": "TokenPocket Token",
+"website": "https://tokenpocket.pro",
+"description": "A leading multi-chain wallet that supports BTC, ETH, BSC, HECO, TRON, EOS, OKExChain, HSC and so on.",
+"explorer": "https://bscscan.com/token/0xa348C8A0E93d97Fbd2629448eCB4CacD6EB2b953",
+"type": "BEP20",
+"symbol": "TPT",
+"decimals": 4,
```
Comment (cbccombine) [inline 0xa348C8A0E93d97Fbd2629448eCB4CacD6EB2b953/info.json:8]: "decimals": 18,

**Comment by opera0001** [comment]
The submitted format is incorrect. Please refer to the tutorial and submit again.

https://tphelp.gitbook.io/cn/wallet-operation/submit-token

**Comment by cbccombine** [comment]
https://github.com/cbccombine/tokens/commit/772eb522c30a5a208c9b11c6ca575ca377633bf0

Hello, it has been re-uploaded, please check, thank you.

combine cbc ***@***.***> 於 2022年3月31日 週四 上午2:56寫道：

**Pull request closed without merging**